.env.example

```diff
@@ -70,6 +70,7 @@ SMART_WALLETS_PORT=5007
 SMART_WALLETS_TCP_PORT=8882
 SMART_WALLETS_MONGO_URI=mongodb://mongo:27017/charge-smart-wallets
 SMART_WALLETS_JWT_SECRET=smart_wallets_jwt_secret
+INCOMING_TOKEN_TRANSFERS_WEBHOOK_ID=put_incoming_token_transfers_webhook_id_here
```

Comment:
do we have this set on production environment?

Reply:
Yes